backend/src/db/queries.ts

```diff
@@ -0,0 +1,88 @@
+import { pool } from './pool';
+import { logger } from '../utils/logger';
+
+const executeQuery = async (
+    text: string,
+    values: string[]
+): Promise<string[]> => {
+    const query = {
+        text: text,
+        values: values,
+    };
+    await pool.connect();
+    const res = await pool.query(query);
+    logger.info(res);
+    await pool.end();
+    // eslint-disable-next-line @typescript-eslint/no-unsafe-return
+    return res.rows;
+};
+
+const selectProjectsbyUserID = async (userID: string) => {
```

Comment:
These should be `export const ...` and contain a return type, since right now all of these will return `any`.